src/endpoint/s3/ops/s3_post_object_select.js

```diff
@@ -86,6 +86,11 @@ async function post_object_select(req, res) {
     const s3select = new S3SelectStream(select_args);
     dbg.log3("select_args = ", select_args);
 
+    s3select.on('error', err => {
+        dbg.error("s3select error:", err, req.path);
```

Comment:
Maybe add "s3select request path" before printing it?
when reading the logs it will be easier to understand the part of the `err` and the part of `req.path`.